documentation/modules/oauth/con-oauth-reauthentication.adoc

```diff
@@ -1,23 +1,41 @@
 // Module included in the following assemblies:
 //
-// assembly-oauth.adoc
+// assembly-oauth-authentication.adoc
 
-[id='con-oauth-reauthentication-{context}']
+[id='{context}']
 = Session re-authentication for Kafka brokers
 
-The Kafka _SASL OAUTHBEARER_ mechanism, which is used for OAuth 2.0 authentication in Strimzi, supports a Kafka feature called the _re-authentication_ mechanism.
+OAuth 2.0 authentication in Strimzi supports _session re-authentication_, which is part of the Kafka Client libraries. 
```

Comment:
Not sure. Is it this: Strimzi's Kafka Client libraries support session re-athentication for OAuth 2.0 authentication?

Reply:
That doesn't sound right, rather it is that the listener on the Kafka broker can be configured with re-authentication (you can turn it on using `maxSecondsWithoutReauthentication` value greater than 0 - by default it is turned off).
But if enabled on the listener it will only work if client libraries support it. Upstream Kafka client libraries support it.

Reply:
That sounds much better to me. @mstruk - FYI